lib/web/ui/user.go

```diff
@@ -30,6 +30,8 @@ type UserListEntry struct {
 	// AuthType is the type of auth service
 	// that the user was authenticated through.
 	AuthType string `json:"authType"`
+	// RoleVariableTraits are traits used for role variables.
+	RoleVariableTraits map[string][]string `json:"roleVariableTraits`
```

Comment:
we already have a field called `trait`, (line 37 of this file), but these traits refer to allowed resource logins/usernames/resourcenames/roles

Reply:
I can't think of a better name, but I'm not sure that this name accurately describes the user's traits here. Maybe LoginTraits or something like that?

Reply:
i just renamed to `allTraits`... is that better? 

```
        // AllTraits returns all the traits.
	// Different from "userTraits" where "userTraits"
	// "selectively" defines traits for local accounts.
	AllTraits map[string][]string `json:"allTraits"`
```

Reply:
as discussed in slack, renamed to `externalTraits`